package.json

```diff
@@ -51,12 +51,13 @@
     "test-proto": "yarn compile-proto && yarn build-proto-roundtrip",
     "unit-core": "yarn mocha core",
     "unit-cli": "yarn mocha --testMatch cli/**/*-test.js",
+    "unit-lantern-trace": "INTERNAL_LANTERN_USE_TRACE=1 yarn mocha core/test/computed/metrics core/test/audits",
     "unit-report": "yarn mocha --testMatch report/**/*-test.js",
     "unit-treemap": "yarn mocha --testMatch treemap/**/*-test.js",
     "unit-viewer": "yarn mocha --testMatch viewer/**/*-test.js",
     "unit-flow": "bash flow-report/test/run-flow-report-tests.sh",
-    "unit": "yarn unit-flow && yarn mocha",
-    "unit:ci": "NODE_OPTIONS=--max-old-space-size=8192 npm run unit",
+    "unit": "yarn unit-flow && yarn mocha && yarn unit-lantern-trace",
+    "unit:ci": "NODE_OPTIONS=--max-old-space-size=8192 npm run unit && NODE_OPTIONS=--max-old-space-size=8192 npm run unit-lantern-trace",
```

Comment:
Could `yarn unit-lantern-trace` be it's own step in CI rather than just grouping it in here. It would make it easier to tell at a glance which version of the unit tests are failing.